Source/WebCore/platform/KeyboardScrollingAnimator.h

```diff
@@ -39,7 +39,7 @@ class KeyboardScrollingAnimator {
 public:
     KeyboardScrollingAnimator(ScrollAnimator&, ScrollingEffectsController&);
 
-    bool beginKeyboardScrollGesture(const PlatformKeyboardEvent&);
+    bool beginKeyboardScrollGesture(const PlatformKeyboardEvent&, Page*);
```

Comment:
It's a layering violation for something in the `/platform` directory to refer to something outside of it (`Page`).